### DISCUSSION TITLE
How can I see to which organisation a shared item belongs?

### DISCUSSION THREAD

**Discussion by nickbe** [dani-garcia/vaultwarden#1752] [Q&A]
Is there a way to see the items organisation somewhere? (WebUI as well as WindowsUI)

**Comment by jjlin** [comment]
I don't think any of the clients show this directly. If you edit the cipher, you can see which collections the cipher belongs to, which can give you a clue about which organization it's associated with.

In the web vault, if you go to the organization vault view for a particular org, you can see the entries that belong to that org.

If you think this is important to have, you can open a feature request at https://community.bitwarden.com/c/feature-requests/5/.

**Reply by nickbe** [reply]
I just saw that similar suggestions were already made in the community. So please close this one. I think there are already others who have the same problems here.

Thanks for pointing ne to the community though. I didn't think to look there. 👍

**Reply by mr-manuel** [reply]
Please upvote for this: https://github.com/dani-garcia/vaultwarden/issues/246#issuecomment-844118167
On Github/Vaultwarden (WebUI) and Bitwarden Forum (Clients)

**Reply by BlackDex** [reply]
Upvoting for it at vaultwarden will not help. At bitwarden forums it could make a difference.
